spec/image.spec.jsx

```diff
@@ -1,9 +1,35 @@
+import React, { createElement } from 'react';
+import { shallow } from 'enzyme';
 import assert from 'power-assert';
 import { renderToString } from 'react-dom/server';
-import { createElement } from 'react';
 import Image from '../src/image';
 
 describe('Image', () => {
+  describe('#componentWillReceiveProps change state on props change', () => {
```

Comment:
- enzyme  本体組み込みの評価の土台として
- React標準のaddonでは setStateがらみの環境を構築するのが煩雑

という理由から、enzyme を実験導入しました。

https://facebook.github.io/react/docs/test-utils.html#createrenderer

> You will need to have window, window.document and window.document.createElement globally available before you import React. Otherwise React will think it can't access the DOM and methods like setState won't work